src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
-export interface Figure {
+enum Shapes {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+enum Colors {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
 
+export interface Figure {
+  shape: Shapes;
+  color: Colors;
+  getArea: () => number;
 }
 
 export class Triangle {
+  shape = Shapes.Triangle;
```

Comment:
```suggestion
  readonly shape = Shapes.Triangle;
```

Reply:
Good point. The shape can be protected from changes.